README.md

```diff
@@ -534,6 +534,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Hamcrest](http://hamcrest.org/JavaHamcrest/) - Matchers that can be combined to create flexible expressions of intent.
 * [JMockit](http://jmockit.org/) - Mocks static, final methods and more.
 * [JUnit](http://junit.org/) - Common testing framework.
+* [JUnitParams](https://pragmatists.github.io/JUnitParams/) - Creation of readable and maintainable parametrised tests.
```

Comment:
Sorry, just took a quick glance. You are right, it is acceptable. No issue using BE or AE.